src/Stryker.ts

```diff
@@ -103,7 +101,7 @@ export default class Stryker {
     }
   }
 
-  private async initialTestRun(inputFiles: InputFile[]): Promise<any> {
+  private async initialTestRun(inputFiles: InputFile[]) {
```

Comment:
What do you mean? The function still has a return type because TypeScript can infer it to be `Promise<{ runResult: RunResult, sandboxCoordinator: SandboxCoordinator}>`, not `Promise<any>`. So, even though we specified it to be `Promise<any>`, TypeScript still knew better than to trust us. We got outsmarted by TypeScript :).

If it it cannot be inferred, we would get an error (result would be implicit `any`). So this is better in my opinion.